lib/http/2/connection.rb

```diff
@@ -496,7 +496,7 @@ def validate_settings(role, settings)
           # allowed frame size (2^24-1 or 16,777,215 octets), inclusive.
           # Values outside this range MUST be treated as a connection error
           # (Section 5.4.1) of type PROTOCOL_ERROR.
-          unless 16_384 <= v && v <= 16_777_215
+          unless v >= 16_384 && v <= 16_777_215
```

Comment:
Maybe using a range here would be easier to read?

Reply:
Yeah, that would probably make things easier to grok. 

Do you know if there is any noticeable perf difference? Microbenchmark to the rescue?